main.c.d

```diff
@@ -0,0 +1,245 @@
+/home/kl0wn/nxdk/samples/zombie-breakout/main.obj: \
```

Comment:
Add:
```
*.obj
*.c.d
*.lib
*.exe
bin/
```

to your .gitignore and remove these files from the repository. They make review much more difficult than it should be.